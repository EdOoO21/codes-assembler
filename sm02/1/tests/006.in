683 -835 -7 561 -465 -925 -1000 -702 357 201 -37 977 555 504 -236 -346 577 -956 -442 1 642 -595 494 786 -153 864 102 104 396 -808 -605 153 133 432 648 492 -457 357 645 249 404 -820 729 -131 -313 -810 -259 638 -161 648 -488 -89 432 -808 547 -597 435 300 631 -404 -801 948 -907 205 848 -589 695 342 869 -262 -4 910 705
