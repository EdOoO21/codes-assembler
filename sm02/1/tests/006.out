705
910
-4
-262
869
342
695
-589
848
205
-907
948
-801
-404
631
300
435
-597
547
-808
432
-89
-488
648
-161
638
-259
-810
-313
-131
729
-820
404
249
645
357
-457
492
648
432
133
153
-605
-808
396
104
102
864
-153
786
494
-595
642
1
-442
-956
577
-346
-236
504
555
977
-37
201
357
-702
-1000
-925
-465
561
-7
-835
683
